designer/dgraph.js

```diff
@@ -0,0 +1,250 @@
+const dgraph = require("dgraph-js");
+const grpc = require("grpc");
+var request = require('request');
+var config_fs_name = './config.json';
+var axios = require('axios')
+var fs = require('fs');
+const bodyParser = require('body-parser');
+var globalConfigFile = require(config_fs_name)
+var config = globalConfigFile.designer;
+config.grpcPort = globalConfigFile.persistent_storage.port;
+config.HostIp = globalConfigFile.external_hostip;
+config.brokerIp=globalConfigFile.coreservice_ip
+config.brokerPort=globalConfigFile.broker.http_port
+
+/*
+   creating grpc client for making connection with dgraph
+*/
+
+function newClientStub() {
+    return new dgraph.DgraphClientStub(config.HostIp+":"+config.grpcPort, grpc.credentials.createInsecure());
+}
+
+// Create a client.
+
+function newClient(clientStub) {
+    return new dgraph.DgraphClient(clientStub);
+}
+
+// Drop All - discard all data and start from a clean slate.
+/*async function dropAll(dgraphClient) {
+    const op = new dgraph.Operation();
+    op.setDropAll(true);
+    await dgraphClient.alter(op);
+}*
+
+/*
+   create scheema for node
+*/
+
+async function setSchema(dgraphClient) {
```

Comment:
[suggestion]all function are now asyn, make sure that all are executed in series so that desired functionality will be achieved.